subprojects/libxmlb.wrap

```diff
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = libxmlb
 url = https://github.com/hughsie/libxmlb.git
-revision = 0.3.2
+revision = 0.3.4
```

Comment:
this means a new dep for snap build according to CI:
```
subprojects/libxmlb/meson.build:119:0: ERROR: Dependency "liblzma" not found, tried pkgconfig
```

Add `liblzma-dev` to build deps and add the library from `liblzma5` to what gets installed (`/lib/x86_64-linux-gnu/liblzma.so.5.2.4`)

Reply:
It seems that libxmlb has some problems with older glib still.  I'd say should hold off bumping that, and fix this in libxmlb and then update the subproject later.

```
[1/603] Compiling C object 'subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-silo-node.c.o'
<command-line>:0:26: warning: "GLIB_VERSION_2_58" is not defined, evaluates to 0 [-Wundef]
/usr/include/glib-2.0/glib/gversionmacros.h:264:45: note: in expansion of macro ‘GLIB_VERSION_MAX_ALLOWED’
 #if !defined (GLIB_VERSION_MAX_ALLOWED) || (GLIB_VERSION_MAX_ALLOWED == 0)
                                             ^~~~~~~~~~~~~~~~~~~~~~~~
[2/603] Compiling C object 'subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-value-bindings.c.o'
<command-line>:0:26: warning: "GLIB_VERSION_2_58" is not defined, evaluates to 0 [-Wundef]
/usr/include/glib-2.0/glib/gversionmacros.h:264:45: note: in expansion of macro ‘GLIB_VERSION_MAX_ALLOWED’
 #if !defined (GLIB_VERSION_MAX_ALLOWED) || (GLIB_VERSION_MAX_ALLOWED == 0)
                                             ^~~~~~~~~~~~~~~~~~~~~~~~
[3/603] Compiling C object 'subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-node.c.o'
FAILED: subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-node.c.o 
cc -Isubprojects/libxmlb/src/554a94d@@xmlb@sha -Isubprojects/libxmlb/src -I../subprojects/libxmlb/src -Isubprojects/libxmlb -I../subprojects/libxmlb -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/root/project/stage/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c99 -g -Wno-aggregate-return -Wunused -Warray-bounds -Wcast-align -Wclobbered -Wdeclaration-after-statement -Wduplicated-branches -Wduplicated-cond -Wempty-body -Wformat=2 -Wformat-nonliteral -Wformat-security -Wformat-signedness -Wignored-qualifiers -Wimplicit-function-declaration -Winit-self -Wlogical-op -Wmissing-declarations -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wmissing-parameter-type -Wmissing-prototypes -Wnested-externs -Wno-error=cpp -Wno-unknown-pragmas -Wno-discarded-qualifiers -Wno-missing-field-initializers -Wno-strict-aliasing -Wno-suggest-attribute=format -Wno-unused-parameter -Wnull-dereference -Wold-style-definition -Woverride-init -Wpointer-arith -Wredundant-decls -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wstrict-prototypes -Wswitch-default -Wtype-limits -Wundef -Wuninitialized -Wunused-but-set-variable -Wunused-variable -Wwrite-strings -fstack-protector-strong -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_46 -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_58 -fPIC -pthread -MD -MQ 'subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-node.c.o' -MF 'subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-node.c.o.d' -o 'subprojects/libxmlb/src/554a94d@@xmlb@sha/xb-node.c.o' -c ../subprojects/libxmlb/src/xb-node.c
<command-line>:0:26: warning: "GLIB_VERSION_2_58" is not defined, evaluates to 0 [-Wundef]
/usr/include/glib-2.0/glib/gversionmacros.h:264:45: note: in expansion of macro ‘GLIB_VERSION_MAX_ALLOWED’
 #if !defined (GLIB_VERSION_MAX_ALLOWED) || (GLIB_VERSION_MAX_ALLOWED == 0)
                                             ^~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/lib/x86_64-linux-gnu/glib-2.0/include/glibconfig.h:9:0,
                 from /usr/include/glib-2.0/glib/gtypes.h:32,
                 from /usr/include/glib-2.0/glib/galloca.h:32,
                 from /usr/include/glib-2.0/glib.h:30,
                 from /usr/include/glib-2.0/gobject/gbinding.h:28,
                 from /usr/include/glib-2.0/glib-object.h:23,
                 from ../subprojects/libxmlb/src/xb-node.c:11:
../subprojects/libxmlb/src/xb-node.c:50:18: warning: implicit declaration of function ‘G_ALIGNOF’ [-Wimplicit-function-declaration]
 G_STATIC_ASSERT (G_ALIGNOF (XbNodeAttrIter) >= G_ALIGNOF (RealAttrIter));
```

Reply:
> fix this in libxmlb and then update the subproject later

agree, I'll do exactly this

Reply:
Done.